<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EzTSPSolver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_task">
       <attribute name="title">
        <string>Task</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView"/>
        </item>
        <item row="0" column="1">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="description_1">
            <property name="text">
             <string>Number of vertices</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_random">
            <property name="text">
             <string>Fill with random values</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="description_2">
            <property name="text">
             <string>Algorithm</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QComboBox" name="comboBox_alghoritm">
            <item>
             <property name="text">
              <string>brute force</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>dynamic programming</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bf multiple visits</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>mst</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>nearest neighbour</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>repetitive nearest neighbour</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>random</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="10" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>18</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_solve">
            <property name="text">
             <string>Solve</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QSpinBox" name="numOfVerticesSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_solution">
       <attribute name="title">
        <string>Solution</string>
       </attribute>
       <widget class="QTextBrowser" name="solutionTextBrowser">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="actionTask"/>
     <addaction name="actionSolution"/>
     <addaction name="actionSolution_and_task"/>
    </widget>
    <addaction name="menuSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_EzTSPSolver"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_EzTSPSolver">
   <property name="text">
    <string>About EzTSPSolver</string>
   </property>
  </action>
  <action name="actionTask">
   <property name="text">
    <string>task</string>
   </property>
  </action>
  <action name="actionSolution">
   <property name="text">
    <string>solution</string>
   </property>
  </action>
  <action name="actionSolution_and_task">
   <property name="text">
    <string>solution and task</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
